src/definitions.rs

```diff
@@ -203,7 +203,7 @@ pub mod api_v2 {
         pub rpc_url: String,
         #[serde(skip_serializing_if = "Option::is_none")]
         pub asset_id: Option<AssetId>,
-        #[serde(skip_serializing)]
```

Comment:
ss58 is always empty at the moment 🤔

Reply:
https://alzymologist.github.io/kalatori-api/#/ as it should be by spec?

Reply:
correct, but it looks like it always empty while should be populated with on chain data. (not for the API but overall)

Reply:
it is not even correct place to store it; ss58 is property of chain, not property of currency.

Reply:
let's remove it altogether maybe? At least let's keep not reporting it, it does not make sense to me at the moment, I've just left it because I might be missing some point here.